app/src/main/java/io/intrepid/skeleton/base/BaseView.java

```diff
@@ -0,0 +1,4 @@
+package io.intrepid.skeleton.base;
```

Comment:
I had an earlier discussion with @gdaniels about empty interfaces.

@f2rf2r Why do you think this `BaseView` interface should be defined, instead of not having one at all.